specification/maps/data-plane/Creator/preview/2022-09-01-preview/wayfind.json

```diff
@@ -5,6 +5,14 @@
     "title": "Azure Maps Wayfinding Service",
     "description": "Azure Maps Wayfinding REST APIs"
   },
+  "x-ms-parameterized-host": {
+    "hostTemplate": "{geography}.atlas.microsoft.com",
```

Comment:
I think you are missing an "endpoint" parameter here.  The "geography" appears to be just "us" or "eu" but the host template should specify the entire "baseUrl" for connecting to the service.

Reply:
I modeled it after all the other files in the same folder: dataset.json, routeset.json, style.json, etc. 
The outcome of dataset, for instance is:  "GET https://{geography}.atlas.microsoft.com/datasets" (see 
https://learn.microsoft.com/en-us/rest/api/maps/v20220901preview/dataset/get?tabs=HTTP) which seems consistent with "hostTemplate": "{geography}.atlas.microsoft.com".